client.go

```diff
@@ -186,6 +187,9 @@ type Client struct {
 	// It allows to queue packets before sending them.
 	// It defaults to 8.
```

Comment:
@aler9 BTW I noticed that this comment is out of with the actual code (which defaults to 256).